uber/site_sections/preregistration.py

```diff
@@ -113,23 +113,26 @@ def form(self, session, message='', edit_id=None, **params):
         """
         params['id'] = 'None'   # security!
 
+        # Both the Attendee class and Group class have identically named
+        # address fields. In order to distinguish the two sets of address
+        # fields in the params, the Group fields are prefixed with "group_"
+        # when the form is submitted. To prevent instantiating the Group object
+        # with the Attendee's address fields, we must clone the params and
+        # rename all the "group_" fields.
+        group_params = dict(params)
```

Comment:
Which pages are those? We would be better off changing those pages to use the group prefix. Having params pull double duty this way is fragile. Like... maybe these are group params, maybe not. Who knows?!